controllers/virtualmachinewebconsolerequest/v1alpha1/webconsolerequest_intg_test.go

```diff
@@ -110,27 +126,59 @@ func webConsoleRequestReconcile() {
 				},
 			}
 			Expect(ctx.Client.Status().Update(ctx, proxySvc)).To(Succeed())
+			Expect(ctx.Client.Create(ctx, vm)).To(Succeed())
+			Expect(ctx.Client.Create(ctx, wcr)).To(Succeed())
 		})
 
-		AfterEach(func() {
+		JustAfterEach(func() {
 			err := ctx.Client.Delete(ctx, wcr)
 			Expect(err == nil || k8serrors.IsNotFound(err)).To(BeTrue())
 			err = ctx.Client.Delete(ctx, vm)
 			Expect(err == nil || k8serrors.IsNotFound(err)).To(BeTrue())
+			err = ctx.Client.Delete(ctx, proxySvc)
+			Expect(err == nil || k8serrors.IsNotFound(err)).To(BeTrue())
+		})
+
+		When("v1a1 provider", func() {
+			It("resource successfully created", func() {
+				Eventually(func() bool {
+					wcr = getWebConsoleRequest(ctx, types.NamespacedName{Name: wcr.Name, Namespace: wcr.Namespace})
```

Comment:
I know this was the prior code, but in case you're not aware there is a nicer way to do assertions in the `Eventually` block: https://onsi.github.io/gomega/#category-3-making-assertions-eminem-the-function-passed-into-codeeventuallycode - so the failure reason is more obvious. I've been trying to clean these up as I'm making other changes in the neighborhood. Up to you if you want to do that here now or not.

Reply:
Thanks. I'll fix it.